pkg/kubelet/volume/iscsi_pd/iscsi_util.go

```diff
@@ -0,0 +1,167 @@
+/*
+Copyright 2015 Red Hat Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package iscsi_pd
+
+import (
+	"errors"
+	"os"
+	"path"
+	"strings"
+	"time"
+
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/kubelet/volume"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/kubelet/volume/gce_pd"
```

Comment:
I don't think details from one plugin should leak into another.

If there are generic mounting utils in GCE that other volumes would benefit from, it is probably better to refactor those commons things into a package all volumes can use.